File: app/channels/group_channel.rb
class GroupChannel < ApplicationCable::Channel
  def subscribed
    # stream_from "some_channel"
  end

  def unsubscribed
    # Any cleanup needed when channel is unsubscribed
  end

  def speak(data)
    binding.pry
    community = Community.create!(content: data['community'])
    template = ApplicationController.renderer.render(partial: 'communitys/community', locals: {community: community})
    ActionCable.server.broadcast 'group_channel', template
  end
end

File: app/javascript/channels/group_channel.js
import consumer from "./consumer"

consumer.subscriptions.create("GroupChannel", {
  connected() {
    // Called when the subscription is ready for use on the server
  },

  disconnected() {
    // Called when the subscription has been terminated by the server
  },

  received(data) {
    const communitys = document.getElementById('messages');
    messages.innerHTML += date
    // Called when there's incoming data on the websocket for this channel
  },

  speak: function(content) {
    return this.perform('speak', {community: 'content'});
  }
});

document.addEventListener('DOMContentLoaded', function() {
  const input = document.getElementById('chat-input')
  const button = document.getElementById('button')
  button.addEventListener('click', function() {
    const content = input.value
    App.group.speak(content)
    input.value = ''
  })
})